lib/segment/src/index/field_index/full_text_index/immutable_inverted_index.rs

```diff
@@ -60,7 +60,7 @@ impl InvertedIndex for ImmutableInvertedIndex {
                     let postings = self.postings.get(idx as usize);
                     hw_counter.incr_delta(
                         size_of::<Option<CompressedPostingList>>()
-                            + postings.map(|i| i.len()).unwrap_or(0),
+                            + postings.map(|i| i.len()).unwrap_or(0) * size_of::<u32>(),
```

Comment:
why do we multiply by `size_of::<u32>()` here?

Reply:
`postings` is of type `Vec<PointOffsetType>`. To calculate the size of `postings`, we have to multiply the length by the size of each item.
Giving it a second thought, we can simply use `size_of_val` inside the `map(..)` function instead